Makefile

```diff
@@ -142,6 +142,28 @@ ifeq ($(UNAME_M),$(filter $(UNAME_M),x86_64 i686 amd64))
 	#CXXFLAGS += -mssse3
 endif
 
+ifneq ($(filter aarch64%,$(UNAME_M)),)
+	# Apple M1, M2, etc.
+	# Raspberry Pi 3, 4, Zero 2 (64-bit)
+	CFLAGS   += -mcpu=native
+	CXXFLAGS += -mcpu=native
```

Comment:
I believe `clang` does not support `-mcpu=native` on most platform. AFAIK only on M1, certain RISC-Vs and if the compiler is manually patched. Maybe we want to detect the compiler before applying flag.

Compiling with the flag fails on compiler explorer.
https://godbolt.org/z/v7868zdG4

Reply:
It was already there.

If you can make a PR to detect clang in this situation, it would be awesome.